### PR TITLE
Add stopwatch script command

### PR DESCRIPTION
**Pull request by pomdtr**
## Description

Add a stopwatch mechanism to raycast. You can start, stop and see progress of an active stopwatch

- [x] New script command

## Screenshot

Not much to see.

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/productivity/stopwatch/start.bash [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# @raycast.icon ⏱
+# @raycast.packageName time
+
+# Documentation:
+# @raycast.description Start a stopwatch
+# @raycast.author Achille Lacoin
+# @raycast.authorURL https://github.com/pomdtr
+
+if [ -f  timestamp_start.txt ]; then
+    echo "A stopwatch already exists!" > /dev/stderr
```
Comment (thomaspaulmann) [inline commands/productivity/stopwatch/start.bash:18]: Good one!

File: commands/productivity/stopwatch/stop.bash [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# @raycast.packageName time
+
+# Documentation:
+# @raycast.description Stop active stopwatch, copy total time
+# @raycast.author Achille Lacoin
+# @raycast.authorURL https://github.com/pomdtr
+
+set -e
+
+PROGRESS=$(./progress.sh)
```
Comment (grzegorzkrukowski) [inline commands/productivity/stopwatch/stop.bash:19]: There is no `progress.sh` file included in this PR - should this be `.bash` instead here?

**Review by grzegorzkrukowski — CHANGES_REQUESTED** [review]

**Comment by grzegorzkrukowski** [comment]
I would also suggest adding `stopwatch-` prefix to all filenames - some users will throw all scripts into one folder.

File: commands/productivity/stopwatch/stop.bash (hunk 2 above)
Comment (pomdtr) [reply commands/productivity/stopwatch/stop.bash:19]: Oups. I'll correct it

Comment (pomdtr) [reply commands/productivity/stopwatch/stop.bash]: Done in da945f7

**Comment by es183923** [comment]
I think putting the file in `/tmp/stopwatch/stopwatch.txt` or somewhere similar would be a better location for the stopwatch temporary file.

**Comment by pomdtr** [comment]
I agree, I moved it to the system TMPDIR in 82d69ea.

Thanks for the feedback!

**Comment by jaklan** [comment]
@pomdtr it's not as issue actually, but `.bash` suffix is pretty rare (even if it's explicitly `bash`, not `sh`, script), I would keep with the standard `.sh` as all other bash scripts in the repository.

**Comment by grzegorzkrukowski** [comment]
@pomdtr why we ended up having both .sh and .bash in this PR ?

**Review by grzegorzkrukowski — CHANGES_REQUESTED** [review]
@pomdtr why we ended up having both .sh and .bash in this PR ? Let's remove duplicated files and leave .sh only

**Review by grzegorzkrukowski — APPROVED** [review]
Looks good now - thanks for the changes

**Pull request merged**